contracts/pcv/PCVGuardian.sol

```diff
@@ -124,6 +124,19 @@ contract PCVGuardian is IPCVGuardian, CoreRefV2 {
         }
     }
 
+    // ---------- PCV Guardian State-Changing API ----------
+
+    // -----------------------------------------------------
+    // -------------------  WARNING!!! ---------------------
+    // -----------------------------------------------------
+    //     USING THESE FUNCTIONS WILL BREAK ACCOUNTING
+    //   IN THE PCV ORACLE. ONLY USE FUNCTIONS IN AN
+    //   EMERGENCY SITUATION IF WITHDRAWING FROM PCV DEPOSITS.
+    // -----------------------------------------------------
+    //   WITHDRAWING FROM A PSM WILL NOT BREAK ACCOUNTING.
```

Comment:
```suggestion
    //   USING THESE FUNCTIONS WILL MAKE THE PCVORACLE THINK
    //   THAT ASSETS FLOWED OUT OF THE SYSTEM, BECAUSE TOKEN
    //   BALANCES ON THE SAFE ADDRESS (DAO TIMELOCK) ARE NOT
    //   COUNTED AS PART OF PCV. ONLY USE FUNCTIONS IN AN
    //   EMERGENCY SITUATION IF WITHDRAWING FROM PCV DEPOSITS.
    // -----------------------------------------------------
    //   WITHDRAWING FROM A PSM WILL NOT HAVE THE SAME
    //   EFFECT BECAUSE ASSETS IN PSM ARE ALREADY COUNTED
    //   OUT OF THE SYSTEM FROM AN ACCOUNTING PERSPECTIVE.
```

I'd rather solve the issue by adding a lens that reads the timelock's balance, but if we go this way, let's have a more thorough warning comment

Reply:
a lens doesn't solve the off chain tracking issue because it won't emit an event when funds are deposited into the timelock. a lens will solve the onchain tracking issue. if we're going to have to write code to fix this, I'd prefer doing things right and moving our setup to an ERC20 Holding PCV Deposit.